frontend/src/pages/history/components/modal/modal.tsx

```diff
@@ -0,0 +1,190 @@
+import styled from '@emotion/styled'
+import {useRef, useState} from 'react'
+
+import {Button} from '@/components/button'
+import {Card} from '@/components/card'
+import {Divider} from '@/components/divider'
+import {Icon} from '@/components/icon'
+import {Rating} from '@/components/rating'
+import {useFetch} from '@/hooks/use-fetch'
+import {API_BASE_URL} from '@/utils/constants'
+
+interface ReviewModalProps {
+  onClose: () => void
+  reservationId: number
+  instructorId: number
+}
+
+interface InstructorResponse {
+  instructorInfo: {
+    id: number
+    name: string
+    phoneNumber: string
+    instructorImage: string
+    pricePerHour: number
+    introduction: string
+    registerDate: string
+    rating: number
+  }
+  academyInfo: {
+    name: string
+    latitude: number
+    longitude: number
+    cert: boolean
+  }
+}
+
+export function ReviewModal({onClose, reservationId, instructorId}: ReviewModalProps) {
+  const {data} = useFetch<InstructorResponse>(`${API_BASE_URL}/instructors/${instructorId}`)
+
+  const [submitting, setSubmitting] = useState<boolean>(false)
+  const [rating, setRating] = useState<number>(5)
```

Comment:
5 라는 숫자가 어떤의미인지 상수로 컴포넌트 밖에 정의해주면 좋겠어요!
의미 파악을 위해 아래 코드까지 살펴보게 되거든요!!

```suggestion
  const DEFAULT_RATING = 5

  const [rating, setRating] = useState<number>(DEFAULT_RATING)
```